homeassistant/components/cert_expiry/manifest.json

```diff
@@ -1,8 +1,9 @@
 {
-  "domain": "cert_expiry",
-  "name": "Cert expiry",
-  "documentation": "https://www.home-assistant.io/components/cert_expiry",
-  "requirements": [],
-  "dependencies": [],
-  "codeowners": []
+    "domain": "cert_expiry",
+    "name": "Cert expiry",
+    "documentation": "https://www.home-assistant.io/components/cert_expiry",
+    "requirements": [],
+    "config_flow": true,
+    "dependencies": [],
+    "codeowners": []
```

Comment:
Do you want to add yourself as codeowner?

Reply:
I did not write this plugin, i only did the config_flow for it so not sure if i should be the coe-owner

Reply:
Well you probably know it better than most of us now.

Reply:
i will add this as last step, doing this now will only result in merge conflicts